pkg/schemas/management.cattle.io/v3/schema.go

```diff
@@ -311,13 +311,13 @@ func authzTypes(schemas *types.Schemas) *types.Schemas {
 		// MustImport(&Version, v3.SetPodSecurityPolicyTemplateInput{}).
 		MustImport(&Version, v3.ImportYamlOutput{}).
 		MustImportAndCustomize(&Version, v3.Project{}, func(schema *types.Schema) {
-			schema.ResourceActions = map[string]types.Action{
-				"setpodsecuritypolicytemplate": {
-					Input:  "setPodSecurityPolicyTemplateInput",
-					Output: "project",
-				},
-				"exportYaml": {},
-			}
+			// schema.ResourceActions = map[string]types.Action{
```

Comment:
Does this need to be commented out or can it be deleted?

Reply:
this is related to psp as well..in this [PR](https://github.com/rancher/shepherd/pull/179/files#diff-b9538a1c8a175ff43d3187a23d5d6b6c488f87d53e9e38a8acba2235688326b2) there is a discussion about the commented code and since those lines were not deleted till now i also commented the code instead of deleting those lines. 
@caliskanugur could you please check once.
Thanks